tests/unit_tests/test_wave_06.py

```diff
@@ -227,9 +228,10 @@ def test_swap_best_by_category_no_other_match_is_false():
     )
 
     assert result == False
-    assert jesse.inventory[0].get_category() != "Electronics"
-    assert jesse.inventory[1].get_category() != "Electronics"
-    assert jesse.inventory[2].get_category() != "Electronics"
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert [item_c, item_b, item_a] == tai.inventory
+    assert [item_f, item_e, item_d] == jesse.inventory
```

Comment:
As above, the list comparisons are suitable since we didn't expect anything to move in the lists. But this also makes the length checks redundant, since checking a list for equality requires the lengths to match (and the compare list has 3).